cmake/projects/glfw/hunter.cmake

```diff
@@ -90,6 +90,28 @@ hunter_add_version(
     533debc50f12ce147d54d210408c7e33654f10a4
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    glfw
+    VERSION
+    "3.3.4-p0"
```

Comment:
Can you remove this version, if it isn't useful?

Reply:
I've deleted the p1 tag, and pointed the p0 tag to the latest glfw commit on the 3.3.4 branch now